keyboards/nuphy/air75_v2/ansi/info.json

```
@@ -0,0 +1,268 @@
+{
+    "keyboard_name": "NuPhy Air75 V2",
+    "manufacturer": "NuPhy",
+    "usb":{
+        "vid": "0x19F5",
+        "pid": "0x3245",
+        "device_version": "0.0.1",
+        "no_startup_check": true
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": true,
+        "command": false,
+        "nkro": true,
+        "key_lock": true,
+        "rgb_matrix": true
+    },
+    "processor": "STM32F072",
+    "bootloader": "stm32-dfu",
+    "debounce": 2,
+    "matrix_pins": {
+        "cols": ["A4", "A5", "A6", "B9", "B0", "B1", "B10", "B11", "B12", "B13", "B14", "B15", "A8", "A9", "A10", "A15", "B3"],
+        "rows": ["C14", "C15", "A0", "A1", "A2", "A3"]
+    },
+    "diode_direction": "COL2ROW",
+    "rgb_matrix": {
+      "driver": "ws2812",
+      "center_point": [80, 30],
+      "max_brightness": 128,
+      "val_steps": 26,
+      "speed_steps": 52,
```

```suggestion
    "rgb_matrix": {
      "default": {
            "animation": "cycle_left_right"
      },
      "driver": "ws2812",
      "center_point": [80, 30],
      "max_brightness": 128,
      "val_steps": 26,
      "speed_steps": 52,
      "sleep": true,
```
This should be done before running `qmk format-json -i keyboards/nuphy/air75_v2/ansi/info.json` to make things look nice.